src/datachain/lib/dc.py

```diff
@@ -252,13 +272,24 @@ def column(self, name: str) -> Column:
         """Returns Column instance with a type if name is found in current schema,
         otherwise raises an exception.
         """
-        name_path = name.split(".")
+        if "." in name:
+            name_path = name.split(".")
+        elif DEFAULT_DELIMITER in name:
+            name_path = name.split(DEFAULT_DELIMITER)
```

Comment:
Just wonder, if it is possible to have both dot (`.`) and `DEFAULT_DELIMITER` (`__`) in column name? 🤔

Reply:
I don't think so. Seems like one is always swapped for the other.